packages/jsii/lib/compiler.ts

```diff
@@ -489,18 +511,18 @@ export interface NonBlockingWatchOptions {
 function _pathOfLibraries(
   host: ts.CompilerHost | ts.WatchCompilerHost<any>,
 ): string[] {
-  if (!COMPILER_OPTIONS.lib || COMPILER_OPTIONS.lib.length === 0) {
+  if (!BASE_COMPILER_OPTIONS.lib || BASE_COMPILER_OPTIONS.lib.length === 0) {
```

Comment:
Does this conditional ever return `true`? `BASE_COMPILER_OPTIONS` is hardcoded by us no?

Reply:
Hmm yeah right... but this may not be hardcoded forever. Since this is just the rename I'd leave it as-is for now.